src/Provider/ShopifyStore.php

```diff
@@ -3,9 +3,12 @@
 namespace Multidimensional\OAuth2\Client\Provider;
 
 use League\OAuth2\Client\Provider\ResourceOwnerInterface;
+use League\OAuth2\Client\Tool\ArrayAccessorTrait;
 
 class ShopifyStore implements ResourceOwnerInterface
 {
+    use ArrayAccessorTrait;
```

Comment:
Thank you. I should have looked at the source code on the ArrayAccessor before approving the commit.